CHANGELOG.md

```diff
@@ -49,46 +49,47 @@ bumped version for float/double range bugfix
 
 ## Benchmarks
 
-### Notes
-
-1. These are **not** percentage (%) increases. Random `Int`s are produced 48.9 times faster!
-2. The only type for which generation is slower is for `Integer`s (on
-   ranges); in the version 1.1 the generation for `Integer` was
-   biased.
-
-### Without Specifying Ranges
-
-    |----------|----------------|----------------|----------------------|
-	| Type     | Cycles/Int 1.1 | Cycles/Int 1.2 | Performance Increase |
-	|----------|----------------|----------------|----------------------|
-	| Ints     |           1508 |          30.84 |                 48.9 |
-	| Word16   |            495 |          30.88 |                 16.0 |
-	| Floats   |           1036 |          35.11 |                 29.5 |
-	| CFloats  |           1054 |          33.75 |                 31.2 |
-	| Doubles  |           1875 |          35.77 |                 52.4 |
-	| CDoubles |            908 |          33.31 |                 27.3 |
-	| Integers |           1578 |          33.09 |                 47.7 |
-	| Bools    |            698 |          36.15 |                 19.3 |
-	| Chars    |            693 |           57.6 |                 12.0 |
-	|----------|----------------|----------------|----------------------|
-
-### Specifying Ranges
-
-	|--------------|----------------|----------------|----------------------|
-	| Type         | Cycles/Int 1.1 | Cycles/Int 1.2 | Performance Increase |
-	|--------------|----------------|----------------|----------------------|
-	| Ints         |            734 |            102 |                  7.2 |
-	| Word16s      |            748 |            115 |                  6.5 |
-	| Floats       |           2055 |          35.88 |                 57.3 |
-	| CFloats      |           1071 |          34.96 |                 30.6 |
-	| Doubles      |           3050 |          35.89 |                 85.0 |
-	| CDoubles     |           1112 |          34.87 |                 31.9 |
-	| Integers     |            534 |            868 |                  0.6 |
-	| Bools        |            739 |          35.22 |                 21.0 |
-	| Chars        |            790 |            133 |                  5.9 |
-	| BIG Integers |         199848 |         103056 |                  1.9 |
-	|--------------|----------------|----------------|----------------------|
-
-
-
+Here are some benchmarks run on a 3.1 GHz Intel Core i7. The full
+benchmarks can be run using e.g. `stack bench`. The benchmarks are
+measured in milliseconds per 100,000 generations. In some cases, the
+performance is over x1000 times better.
```

Comment:
Same here - I give much more weight to such claims personally when a lower bound is also given.